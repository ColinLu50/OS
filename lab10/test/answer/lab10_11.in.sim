5178
174
176
147
174
146
