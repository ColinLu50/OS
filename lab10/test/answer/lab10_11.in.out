FCFS
26 -> 103 -> 118 -> 57 -> 132 -> 10 -> 49 -> 51 -> 4 -> 1 -> 115 -> 71 -> 144 -> 25 -> 78 -> 55 -> 144 -> 119 -> 127 -> 56 -> 114 -> 47 -> 73 -> 41 -> 2 -> 130 -> 72 -> 77 -> 111 -> 72 -> 64 -> 26 -> 138 -> 31 -> 70 -> 29 -> 52 -> 22 -> 66 -> 150 -> 88 -> 69 -> 54 -> 29 -> 47 -> 101 -> 34 -> 76 -> 111 -> 139 -> 142 -> 40 -> 58 -> 122 -> 143 -> 100 -> 114 -> 110 -> 93 -> 15 -> 140 -> 59 -> 63 -> 3 -> 7 -> 86 -> 136 -> 134 -> 83 -> 38 -> 136 -> 118 -> 108 -> 50 -> 112 -> 135 -> 144 -> 136 -> 88 -> 105 -> 68 -> 20 -> 71 -> 36 -> 37 -> 46 -> 69 -> 148 -> 21 -> 71 -> 124 -> 5 -> 112 -> 46 -> 78 -> 129 -> 47 -> 48 -> 35 -> 139 -> 95 -> 47 -> 91 -> 88 -> 117 -> 17 -> 31 -> 121 -> 136 -> 86
5178
SSTF
26 -> 26 -> 25 -> 22 -> 21 -> 20 -> 17 -> 15 -> 10 -> 7 -> 5 -> 4 -> 3 -> 2 -> 1 -> 29 -> 29 -> 31 -> 31 -> 34 -> 35 -> 36 -> 37 -> 38 -> 40 -> 41 -> 46 -> 46 -> 47 -> 47 -> 47 -> 47 -> 48 -> 49 -> 50 -> 51 -> 52 -> 54 -> 55 -> 56 -> 57 -> 58 -> 59 -> 63 -> 64 -> 66 -> 68 -> 69 -> 69 -> 70 -> 71 -> 71 -> 71 -> 72 -> 72 -> 73 -> 76 -> 77 -> 78 -> 78 -> 83 -> 86 -> 86 -> 88 -> 88 -> 88 -> 91 -> 93 -> 95 -> 100 -> 101 -> 103 -> 105 -> 108 -> 110 -> 111 -> 111 -> 112 -> 112 -> 114 -> 114 -> 115 -> 117 -> 118 -> 118 -> 119 -> 121 -> 122 -> 124 -> 127 -> 129 -> 130 -> 132 -> 134 -> 135 -> 136 -> 136 -> 136 -> 136 -> 138 -> 139 -> 139 -> 140 -> 142 -> 143 -> 144 -> 144 -> 144 -> 148 -> 150
174
SCAN
26 -> 25 -> 22 -> 21 -> 20 -> 17 -> 15 -> 10 -> 7 -> 5 -> 4 -> 3 -> 2 -> 1 -> 0 -> 29 -> 31 -> 34 -> 35 -> 36 -> 37 -> 38 -> 40 -> 41 -> 46 -> 47 -> 48 -> 49 -> 50 -> 51 -> 52 -> 54 -> 55 -> 56 -> 57 -> 58 -> 59 -> 63 -> 64 -> 66 -> 68 -> 69 -> 70 -> 71 -> 72 -> 73 -> 76 -> 77 -> 78 -> 83 -> 86 -> 88 -> 91 -> 93 -> 95 -> 100 -> 101 -> 103 -> 105 -> 108 -> 110 -> 111 -> 112 -> 114 -> 115 -> 117 -> 118 -> 119 -> 121 -> 122 -> 124 -> 127 -> 129 -> 130 -> 132 -> 134 -> 135 -> 136 -> 138 -> 139 -> 140 -> 142 -> 143 -> 144 -> 148 -> 150
176
C-SCAN
26 -> 25 -> 22 -> 21 -> 20 -> 17 -> 15 -> 10 -> 7 -> 5 -> 4 -> 3 -> 2 -> 1 -> 0 -> 150 -> 150 -> 148 -> 144 -> 143 -> 142 -> 140 -> 139 -> 138 -> 136 -> 135 -> 134 -> 132 -> 130 -> 129 -> 127 -> 124 -> 122 -> 121 -> 119 -> 118 -> 117 -> 115 -> 114 -> 112 -> 111 -> 110 -> 108 -> 105 -> 103 -> 101 -> 100 -> 95 -> 93 -> 91 -> 88 -> 86 -> 83 -> 78 -> 77 -> 76 -> 73 -> 72 -> 71 -> 70 -> 69 -> 68 -> 66 -> 64 -> 63 -> 59 -> 58 -> 57 -> 56 -> 55 -> 54 -> 52 -> 51 -> 50 -> 49 -> 48 -> 47 -> 46 -> 41 -> 40 -> 38 -> 37 -> 36 -> 35 -> 34 -> 31 -> 29
147
LOOK
26 -> 25 -> 22 -> 21 -> 20 -> 17 -> 15 -> 10 -> 7 -> 5 -> 4 -> 3 -> 2 -> 1 -> 29 -> 31 -> 34 -> 35 -> 36 -> 37 -> 38 -> 40 -> 41 -> 46 -> 47 -> 48 -> 49 -> 50 -> 51 -> 52 -> 54 -> 55 -> 56 -> 57 -> 58 -> 59 -> 63 -> 64 -> 66 -> 68 -> 69 -> 70 -> 71 -> 72 -> 73 -> 76 -> 77 -> 78 -> 83 -> 86 -> 88 -> 91 -> 93 -> 95 -> 100 -> 101 -> 103 -> 105 -> 108 -> 110 -> 111 -> 112 -> 114 -> 115 -> 117 -> 118 -> 119 -> 121 -> 122 -> 124 -> 127 -> 129 -> 130 -> 132 -> 134 -> 135 -> 136 -> 138 -> 139 -> 140 -> 142 -> 143 -> 144 -> 148 -> 150
174
C-LOOK
26 -> 25 -> 22 -> 21 -> 20 -> 17 -> 15 -> 10 -> 7 -> 5 -> 4 -> 3 -> 2 -> 1 -> 150 -> 148 -> 144 -> 143 -> 142 -> 140 -> 139 -> 138 -> 136 -> 135 -> 134 -> 132 -> 130 -> 129 -> 127 -> 124 -> 122 -> 121 -> 119 -> 118 -> 117 -> 115 -> 114 -> 112 -> 111 -> 110 -> 108 -> 105 -> 103 -> 101 -> 100 -> 95 -> 93 -> 91 -> 88 -> 86 -> 83 -> 78 -> 77 -> 76 -> 73 -> 72 -> 71 -> 70 -> 69 -> 68 -> 66 -> 64 -> 63 -> 59 -> 58 -> 57 -> 56 -> 55 -> 54 -> 52 -> 51 -> 50 -> 49 -> 48 -> 47 -> 46 -> 41 -> 40 -> 38 -> 37 -> 36 -> 35 -> 34 -> 31 -> 29
146
